packages/xstate-wallet/src/integration-tests/funding.test.ts

```diff
@@ -34,8 +34,7 @@ it('allows for two wallets to fund an app', async () => {
   const createEvent = generateCreateChannelRequest(playerA.participant, playerB.participant);
   const createPromise = playerA.messagingService.outboxFeed
     .pipe(
-      filter(m => 'id' in m && m.id === createEvent.id),
-      map(m => m as CreateChannelResponse),
+      filter((m): m is CreateChannelResponse => 'id' in m && m.id === createEvent.id),
```

Comment:
Why `is`? Just curious the use case

Reply:
It serves as a type guard, which `pipe` can use to narrow the type later down the pipe. In this case, it serves to let [this](https://github.com/statechannels/monorepo/blob/4020bc8e988687d11ac5cee4b84ad3250fa32908/packages/xstate-wallet/src/integration-tests/funding.test.ts#L46) compile.